### PR TITLE
fix: modifed content card styles to meet WCAG 2.1 AA

### LINKED ISSUE #549: Content card fails WCAG 2.1 AA
**Describe the bug**
No hover state for clickable content cards

Needs to be visibly different when the user is hovering over the card.

Example: https://service-manual.nhs.uk

### PR DESCRIPTION
**Pull request by houbly**
fixes #549

### CODE REVIEW

**Review by catiarodriguescosta — APPROVED** [review]
👍

File: src/wmnds/components/content-card/_content-card.scss [hunk 1]
```diff
@@ -1,3 +1,15 @@
+//accessibility styles
+@mixin accessibility-hover {
+  h2 {
+    text-decoration: underline;
```
Comment (daylesalmon) [inline src/wmnds/components/content-card/_content-card.scss:4]: This should only happen when hovered according to: https://github.com/wmcadigital/wmn-design-system/issues/549#issuecomment-753901729

**Review by daylesalmon — CHANGES_REQUESTED** [review]

**Review by daylesalmon — APPROVED** [review]

File: src/wmnds/components/content-card/_content-card.scss (hunk 1 above)
Comment (daylesalmon) [reply src/wmnds/components/content-card/_content-card.scss:4]: None-issue...Resolved by speaking with @houbly.

**Pull request merged**